azure-iot-hub-devicesdk/README.md

```diff
@@ -1,111 +1,45 @@
+# Azure IoT Hub Device SDK
 
-Azure IoT Hub Device SDK
-========================
+The Azure IoT Hub Device SDK for Python provides clients for both Devices and Modules to communicate with the Azure IoT Hub.
 
-## Install
+**Note that this SDK is currently in preview, and is subject to change.**
 
-We currently do not provide a binary distribution of our package, which means you'll have to clone the repository.
-Once you've cloned the repository, please run the `env_setup.py` script to setup the environment and to be able to run
-the samples.
+## Features
+The SDK provides the following clients:
+### Device Client
+* Send telemetry messages to Azure IoT Hub
+* Receive Cloud-to-Device (C2D) messages from the Azure IoT Hub
+### Module Client
+* Supports Azure IoT Edge Hub and Azure IoT Hub
+* Send telemetry messages to a Hub or to another Module
+* Receive Input messages from a Hub or other Modules
 
-## Quick start
+These clients are synchronous and communicate using the MQTT protocol
 
+## Installation
+We currently do not provide a binary distribution of our package, which means you'll have to clone the [repository](https://github.com/Azure/azure-iot-sdk-python-preview) and install with:
```

Comment:
I assume this must be run from the package directory itself? can we call out where this must be run from for each SDK?

Reply:
It's meant to be in the same directory as this README since the setup.py is right there. But I can add clarification.